extensions/sql-migration/src/models/migrationWizardPage.ts

```diff
@@ -80,4 +81,55 @@ export abstract class MigrationWizardPage {
 		const current = this.wizard.currentPage;
 		await this.wizard.setCurrentPage(current + 1);
 	}
+
+	protected showDialogMessage(
```

Comment:
is this showDialogMessage as the one in tabBase.ts, maybe combine the two implementations into a single method in utils.ts